src/main/kotlin/com/hedvig/rapio/qa/QaMemberServiceClient.kt

```diff
@@ -0,0 +1,20 @@
+package com.hedvig.rapio.qa
+
+import com.hedvig.rapio.externalservices.memberService.dto.IsMemberRequest
+import com.hedvig.rapio.qa.dto.MemberServiceUnsignMemberRequest
+import com.hedvig.rapio.qa.dto.UnsignMemberRequest
+import org.springframework.cloud.openfeign.FeignClient
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+
+@FeignClient(
+    name = "member-service",
+    url = "\${hedvig.member-service.url:member-service}"
+)
+interface QaMemberServiceClient {
```

Comment:
I'm not the biggest fan of "Qa" but I honestly don't know of a better name, this is just a bad suggestion
```suggestion
interface StagingToolsMemberServiceClient {
```